design/sd-local.md

```diff
@@ -135,6 +135,7 @@ $ sdlocal build [job-name] [options]
 - `--artifacts-dir [path]` Path to the host side directory which is mounted into `$SD_ARTIFACTS_DIR`. (default: `./sd-artifacts`)
 - `-m, --memory [size]` Set memory size which Build Container can use. Either b, k, m, g can be used as a size unit. (default: ?)
 - `--src-url [repository url]` Set repository URL which is to build when user use the remote repository without local files.
+- `--always-pull-disable` Disable `sd-local` from always pulling build image.
```

Comment:
can we call it `--disable-image-pull` ?

Reply:
I fixed it. It seems better than before. thx